coffea/jetmet_tools/CorrectedJetsFactory.py

```diff
@@ -171,6 +191,29 @@ def build(self, jets, lazy_cache):
         out_dict = dict(in_dict)
 
         # take care of nominal JEC (no JER if available)
+        ## General setup to use correction-lib
+        clib_json = self.jec_year
+        json_path = f"/cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/JME/{clib_json}/jet_jerc.json.gz"
```

Comment:
we do not put hard dependencies on cvmfs paths in coffea.

Reply:
This should be configurable.

I'd also be careful of building new objects every time you call `uncertainties`, this can become very slow.

Reply:
Also, the json file name would change depending on if it's AK8 or AK4. So, I would also recommend against hard-coding this and instead keeping this in the jec_stack.